libs/service-portal/core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useLocale } from '@island.is/localization'
+import { MessageDescriptor } from 'react-intl'
+import { Text, Box } from '@island.is/island-ui/core'
+
+interface Props {
+  title: MessageDescriptor
+  description?: MessageDescriptor
+}
+
+export const EmptyState = ({ title, description }: Props) => {
+  const { formatMessage } = useLocale()
+  return (
+    <>
+      <Text marginBottom={1} variant="h3">
+        {formatMessage(title)}
+      </Text>
+      {description && <Text>{formatMessage(description)}</Text>}
+      <Box
+        component="img"
+        marginTop={4}
+        width="full"
+        src="/assets/images/emptyState.svg"
```

Comment:
This url will probably have to be prefixed with /minarsidur so it works on live, see Greeting.treat.ts

Reply:
Looks like this won't work locally. Do we have a way for this to work locally and live? @atligudl